src/main/java/core/basesyntax/dao/FruitStorageDao.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.dao;
+
+import java.util.Set;
+
+public interface FruitStorageDao {
+    void add(String fruit, int quantity);
+
+    int getQuantity(String fruit);
+
+    Set<String> getFruitSet();
```

Comment:
```suggestion
    Map.Entry<String, Integer> getAll();
```

Reply:
@d1sam Can we return here just a Map? After that our getAll can be used for iterating and processed in generator report method.